cmd/kyma/upgrade/cmd.go

```diff
@@ -34,15 +34,15 @@ func NewCmd(o *Options) *cobra.Command {
 	cobraCmd := &cobra.Command{
 		Use:   "upgrade",
 		Short: "Upgrades Kyma",
-		Long:  `Use this command to upgrade the Kyma version on a cluster.`,
+		Long:  `Use this command to upgrade the Kyma version on a cluster. During upgrade, you can set one of the predefined [profiles](https://kyma-project.io/docs/master/root/kyma/#installation-overview-profiles) for your Kyma deployment. If the ` + "`--profile`" + ` flag is not specified, Kyma is deployed with the default chart values.`,
```

Comment:
Please do not use markdown notation. The main purpose of this documentation is to show it when using the CLI on a terminal.

Reply:
I removed this part from both `kyma install` and `kyma upgrade` command.